### DISCUSSION TITLE
InstantSearchNext potential issues with CORS

### DISCUSSION THREAD

**Discussion by gyto** [algolia/instantsearch#5888] [General]
I might have found an issue when using InstantSearchNext with client-side rendering to a remote server which caused potential Cors issues. Developing on a local machine did not cause any issues however, as I switched the service to production, it fired cors to `Access-Control-Allow-Origin` (the server has a generic cors setup with allows all origins).

I did not want to point out the red flag in /issues because the feature is still experimental, so I am asking around if anyone has gone down this type of rabbit hole.

**Comment by Haroenv** [comment]
Thanks for your report! do you have a reproduction of this issue somewhere?
